docs/UserGuide.md

```diff
@@ -164,7 +168,7 @@ Examples:
 
 :information_source:
  * The default value for `PROTEIN`, `FATS` and `CARBS` is 0. The default date is the day in which the food item is added.<br>
- * Values of `PROTEIN`, `FATS` and `CARBS` must be a non-negative integer.<br>
+ * Values of `PROTEIN`, `FATS` and `CARBS` must be a non-negative integer, not exceeding 1000g.<br>
```

Comment:
Its less than 1000 rather than not exceeding

Reply:
Alright sure!